tasks/certificate.yml

```
@@ -5,5 +5,5 @@
   register: letsencrypt_cert
 
 - name: Generate new certificate if one doesn't exist
-  shell: "certbot certonly --nginx --email '{{ letsencrypt_email }}' --agree-tos -d '{{ domain_name }}' {% if certbot_nginx_cert_name is defined %} --cert-name '{{ certbot_nginx_cert_name }}' {% endif %} {% if letsencrypt_staging %} --staging {% endif %}"
+  shell: "certbot certonly --nginx --email '{{ letsencrypt_email }}' --agree-tos -d '{{ domain_name }}' {% if certbot_nginx_cert_name is defined %} --cert-name '{{ certbot_nginx_cert_name }}' {% endif %} {% if letsencrypt_staging %} --staging {% endif %}"  # noqa 204
```

What if we split the line?

```suggestion
  shell: >
    "certbot certonly --nginx --email '{{ letsencrypt_email }}'
    --agree-tos -d '{{ domain_name }}'
    {% if certbot_nginx_cert_name is defined %}
    --cert-name '{{ certbot_nginx_cert_name }}'
    {% endif %}
    {% if letsencrypt_staging %} --staging {% endif %}"
```

Would this work? Do you like it? [Source](https://robertdebock.nl/2018/11/01/lines-should-be-no-longer-than-120-chars.html)

---

Yep, I prefer @raneq 's proposal.

---

Applied!